src/main/java/core/basesyntax/service/serviceimpl/FileWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, File file) {
+        try (BufferedWriter bufferedWriter
+                     = new BufferedWriter(new java.io.FileWriter(file))) {
```

Comment:
It is good practice to use the try-with-resources statement to ensure that resources are closed properly, which you have done correctly here.